azure/v11.4.0/README.md

```diff
@@ -69,4 +69,5 @@ since [v1.16.8](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/C
 - Changed NGINX Service type from NodePort to LoadBalancer for Azure.
 - Made NGINX LoadBalancer Service external traffic policy configurable.
 - Use Local as NGINX LoadBalancer Service default external traffic policy.
+- Supported configuring (via `controller.service.public` configuration property) whether NGINX LoadBalancer Service should be public (default) or internal.
```

Comment:
@sslavic is this clear in our nginx docs? Would it be helpful to document it?

Reply:
Yes, every config option is documented in values.yaml, which is rendered in happa through readme